style.css

```diff
@@ -0,0 +1,228 @@
+/* My body config */
+body {
+  margin: 0;
+  padding: 0;
+  background: white;
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  height: auto;
+}
+
+/* initialize the .header style: sizes, colors and fonts */
+.header {
+  padding-top: 1px;
+  height: 682px;
+  margin: 0;
+  color: white;
+  background-color: black;
+  width: 100%;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+/* using flexbox for nav tag */
+.nav {
+  margin-bottom: -100px;
+  width: 100%;
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #3c3a39;
+}
```

Comment:
- [ ] Great work positioning your header! 💯 However, please kindly consider adding the property top: 0 to it. Currently, you have a small 1px space separating your header from the top of the page. This may look like an insignificant detail, but we want to make sure recruiters see our most professional work when they check our portfolio. 🤓

This is your design 👇🏽

![design_2](https://user-images.githubusercontent.com/116701485/235016750-f48f34c6-4aa3-46b7-b760-cb096b3901ee.png)

This is the Figma design 👇🏽

![figma_2](https://user-images.githubusercontent.com/116701485/235016584-4222ea38-ed67-47e7-8cc6-2880c9056876.png)